packages/terra-base/src/Base.jsx

```diff
@@ -73,20 +75,25 @@ class Base extends React.Component {
       children,
       locale,
       customMessages,
+      strictMode,
       translationsLoadingPlaceholder,
       ...customProps
     } = this.props;
 
     const messages = Object.assign({}, this.state.messages, customMessages);
+    const renderStrictMode = strictMode ? (<React.StrictMode>{children}</React.StrictMode>) : children;
```

Comment:
Should we add a jest test for this scenario?

Reply:
Updated: https://github.com/cerner/terra-core/pull/1890/commits/ff2991ba0e7d0e67067af930892a713b2fded5b1